pom.xml

```diff
@@ -76,11 +76,6 @@
             <artifactId>antlr4-runtime</artifactId>
             <version>4.13.1</version>
         </dependency>
```

Comment:
As discussed internally, we should revert the changes to the dependencies

Reply:
I agree. But keep in mind, that there currently is a bug in the uvl-parser version in the maven repo.